ldms/src/ldmsd/ldmsd_request.c

```diff
@@ -6012,50 +6009,38 @@ static int stream_publish_handler(ldmsd_req_ctxt_t reqc)
 		goto err_reply;
 	}
 
+	reqc->errcode = 0;
+	ldmsd_send_req_response(reqc, "ACK");
```

Comment:
LDMSD will send two responses back to the requester if no data is provided. Should we validate the stream before sending the ACK response (we won't validate if the JSON-formated string can be parsed or not)?

Reply:
The purpose of the change is to reduce the overhead of streams data if it is only being forwarded and not delivered. If we validate the data, ldmsd will incur the CPU hit even it is not delivering the data to any client. Good catch on the dup response. I'll fix it.